recOrder/plugin/workers/acquisition_workers.py

```diff
@@ -906,6 +901,54 @@ def work(self):
         self.phase_image_emitter.emit(phase)
         self.meta_emitter.emit(meta)
 
+    def _check_exposure(self) -> None:
+        """
+        Check that all LF channels have the same exposure settings. If not, use the State0 exposure.
+
+        Parameters
+        ----------
+        settings:       (json) JSON dictionary conforming to MM SequenceSettings
+        """
+        logging.debug('Verifying exposure times...')
+        # parse exposure times
+        channel_exposures = []
+        for _, channel in enumerate(self.settings['channels']):
+            channel_exposures.append(channel['exposure'])
+
+        channel_exposures = np.array(channel_exposures)
+        if not np.all(channel_exposures == channel_exposures[0]):
+            # warn user that not all channels exposures were the same
+            logging.warning('The exposure times of each State are not equal!\nAcquiring with the exposure of State0...')
```

Comment:
I think you mentioned you're looking into a napari-level warning? If so I think this is great.

Reply:
Will look into that with future commits